jf_agent/validation.py

```diff
@@ -200,3 +245,19 @@ def validate_memory(config):
     except Exception as e:
         print(f"  ERROR: Could not obtain memory and/or disk usage information. {e}")
         return False
+
+
+def upload_to_s3(jellyfish_api_base: str, jellyfish_api_token: str, healthcheck_result: IngestionHealthCheckResult) -> None:
```

Comment:
NIT: this technically isn't uploading to S3, it's uploading to jellyfish to then go to s3, so maybe `submit_health_check_to_jellyfish` would be a better name. That's pretty pedantic of me tho 😜